web-store/src/main/resources/static/Rest.txt

```diff
@@ -0,0 +1,189 @@
+Models
+
+User {
+  "id": String,
+  "username": String,
+  "firstName": String,
+  "lastName": String,
+  "email": String,
+  "password": String,
+  "phone": NUMBER,
+}
+
+Item { 
+  "id": String,
+  "title": String,
+  "cost": Integer,
+  "weight": Double,
+  "manufacturer": String
+}
+
+
+

```

Comment:
+ done
- it does not find in your file

Enities:
+ user:
	+ create
	+ read by id
		+ 404 not found
	+ read all
	+ update
	+ partially update
	+ delete
 - helmet ?
	- create ?
	- read by id ?
	- read all (filter + pagination) ?
	+ update +
	- read all ?
	- update ?
	- delete ?
 - subresource ?
 - working with order ?